editor/editor_help.cpp

```diff
@@ -245,6 +246,13 @@ void EditorHelp::_add_type(const String &p_type, const String &p_enum) {
 			class_desc->add_text("Array");
 			class_desc->pop();
 			class_desc->add_text("[");
+		} else if (is_bitfield) {
+			class_desc->push_color(Color(theme_cache.type_color, 0.5));
+			class_desc->push_hint(TTR("This value is a bitmask of the following flags."));
```

Comment:
```suggestion
			class_desc->push_hint(TTR("This value is integer composed as a bitmask of the following flags."));
```
Not sure that's the best phrasing, but I think we should make it explicit it's an `int`.